solidity/scripts/ScriptingLibrary/ScriptingLibrary.sol

```diff
@@ -6,7 +6,9 @@ import {XERC20Factory, IXERC20Factory} from '../../contracts/XERC20Factory.sol';
 import {Script} from 'forge-std/Script.sol';
 
 contract ScriptingLibrary {
```

Comment:
`ScriptingLibrary` can be extended from some third party library with the same functions?.

Reply:
probably yes, but this was already there and we didn't meant to change it. blame `forge fmt`